tui2/pages/progress/progress.go

```diff
@@ -105,4 +117,5 @@ func (p *Progress) SetSize(w, h int) {
 	headerHeight := 7
 	p.Common.SetSize(w, h)
 	p.bars.SetSize(w, h-headerHeight)
+	p.Styles.StatusBarValue.Width(p.Common.Width - labelsMaxLen() - 2) // adjust width to force word wrap
```

Comment:
Seems good, can you just document the reason for the `-2`.

Update the CHANGELOG.md file and we are good to go.

Reply:
`2` is for padding. Moved it to the `styles.go` since it doesn't change.
Updated changelog.